src/test/java/underdogs/devbie/answer/controller/AnswerControllerTest.java

```diff
@@ -0,0 +1,192 @@
+import static org.mockito.BDDMockito.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static underdogs.devbie.auth.controller.AuthControllerTest.*;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MvcResult;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import underdogs.devbie.MvcTest;
+import underdogs.devbie.answer.domain.Answer;
+import underdogs.devbie.answer.domain.AnswerContent;
+import underdogs.devbie.answer.domain.Answers;
+import underdogs.devbie.answer.dto.AnswerCreateRequest;
+import underdogs.devbie.answer.dto.AnswerResponse;
+import underdogs.devbie.answer.dto.AnswerResponses;
+import underdogs.devbie.answer.dto.AnswerUpdateRequest;
+import underdogs.devbie.answer.service.AnswerService;
+import underdogs.devbie.auth.controller.interceptor.BearerAuthInterceptor;
+import underdogs.devbie.auth.controller.resolver.LoginUserArgumentResolver;
+import underdogs.devbie.user.domain.User;
+
+@WebMvcTest(controllers = AnswerController.class)
+public class AnswerControllerTest extends MvcTest {
+
+    public static final String TEST_ANSWER_CONTENT = "TEST_ANSWER_CONTENT";
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @MockBean
+    private BearerAuthInterceptor bearerAuthInterceptor;
+
+    @MockBean
+    private LoginUserArgumentResolver loginUserArgumentResolver;
+
+    @MockBean
+    private AnswerService answerService;
+
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = User.builder()
+            .id(1L)
+            .oauthId("TEST_USER")
+            .email("TEST_EMAIL")
+            .build();
+        given(bearerAuthInterceptor.preHandle(any(), any(), any())).willReturn(true);
+        given(loginUserArgumentResolver.resolveArgument(any(), any(), any(), any())).willReturn(user);
+        given(loginUserArgumentResolver.supportsParameter(any())).willReturn(true);
+    }
+
+    @DisplayName("사용자 요청을 받아 Answer 생성")
+    @Test
+    void save() throws Exception {
+        AnswerCreateRequest answerCreateRequest = AnswerCreateRequest.of(1L, TEST_ANSWER_CONTENT);
+
+        String inputString = OBJECT_MAPPER.writeValueAsString(answerCreateRequest);
+        given(answerService.save(user, answerCreateRequest)).willReturn(1L);
+
+        postAction("/api/answers", inputString, TEST_TOKEN)
+            .andExpect(status().isCreated())
+            .andDo(print());
+    }
+
+    @DisplayName("사용자 요청을 받아 Answer 저장 시 예외 발생 - 유효 하지 않은 Question Id")
+    @Test
+    void save2() throws Exception {
```

Comment:
@DisplayName으로 명시하고 있지만 테스트코드명도 구체적으로 써주는게 좋을 것 같아요!

```suggestion
    void saveWithInvalidQuestionId_ShouldThrowBadRequestException() throws Exception {
```